src/jit/function-builder.cc

```diff
@@ -1762,6 +1839,41 @@ bool FunctionBuilder::Emit(TR::BytecodeBuilder* b,
 //    UNSIGNED TYPE NOT HANDLED
 //    case Opcode::I64TruncF64U:
 //      EmitTruncation<uint64_t, double>(b, pc, &stack);
+//      break;
+
+    case Opcode::I32TruncSatF32S:
+      EmitSaturatingTruncation<int32_t, float>(b, &stack);
+      break;
+
+    case Opcode::I32TruncSatF32U:
+      EmitUnsignedSaturatingTruncation<uint32_t, float>(b, &stack);
+      break;
+
+    case Opcode::I32TruncSatF64S:
+      EmitSaturatingTruncation<int32_t, double>(b, &stack);
+      break;
+
+
+    case Opcode::I32TruncSatF64U:
+      EmitUnsignedSaturatingTruncation<uint32_t, double>(b, &stack);
+      break;
+
+    case Opcode::I64TruncSatF32S:
+      EmitSaturatingTruncation<int64_t, float>(b, &stack);
+      break;
+
+//    UNSIGNED TYPE NOT HANDLED
+//    case Opcode::I64TruncSatF32U:
+//      EmitSaturatingTruncation<uint64_t, float>(b, &stack);
```

Comment:
I think this should be

```suggestion
//      EmitUnsignedSaturatingTruncation<uint64_t, float>(b, &stack);
```

Reply:
The special "unsigned" variants of the truncation functions can only handle truncation to unsigned 32-bit integers, and I think the idea was that the special variants would be removed by the time 64-bit unsigned truncation was supported.

Regardless, the comments here are consistent with the existing comments for `I64TruncF32U` and `I64TruncF64U` above this, which do not use the unsigned variant.